proto/buf/alpha/registry/v1alpha1/veracity.proto

```diff
@@ -0,0 +1,23 @@
+// Copyright 2020-2022 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package buf.alpha.registry.v1alpha1;
+
+enum Veracity {
```

Comment:
Needs docs on what this is. Also is veracity the right word here?

Reply:
I was trying to find the most accurate word for this, as `verification` doesn't seems accurate enough? any thoughts?

Reply:
maybe ["provenance"](https://en.wiktionary.org/wiki/provenance)?

> 1. Place or source of origin.
> 4. ([computing](https://en.wiktionary.org/wiki/computing#Noun)) The copy history of a piece of data, or the intermediate pieces of data utilized to compute a final data element, as in a database record or web site (data provenance)

similar to "veracity" though, I don't think it's a common word. Maybe just "verification status" for simplicity?

Reply:
updated to `VerificationStatus` as I think it's the easiest one to understand for now, but happy to change if we can come up with a better name